maven-embedder/src/test/java/org/apache/maven/cli/event/ExecutionEventLoggerTest.java

```diff
@@ -73,6 +83,7 @@ public void testProjectStarted()
         inOrder.verify( logger ).info( "" );
         inOrder.verify( logger ).info( "------------------< org.apache.maven:maven-embedder >-------------------" );
         inOrder.verify( logger ).info( "Building Apache Maven Embedder 3.5.4-SNAPSHOT" );
+        inOrder.verify( logger ).info( "    maven-embedder/pom.xml" );
```

Comment:
Seems to be option 1, therefore I would expect rather:
```
Building XXX...
    from .../pom.xml
```

Side note: We have inconsistent indentation. Some use two, some four spaces.

Reply:
I love "Building xxx \n from yyy"
and it adds a clear reason to indentation to match "Building" length, not taste (that inherently varies between people)